src/main.ts

```diff
@@ -21,12 +21,7 @@ async function run() {
     try {
 
         // set up auth/environment
-        const token = process.env['GITHUB_TOKEN']
-        if (!token) {
-            throw new Error(
-                `No GitHub token found`
-            )
-        }
+        const token = process.env['GITHUB_TOKEN'] || core.getInput("token")
```

Comment:
For backwards compatibility the environment variable is used if available. If not it falls back to the `token` input.